docs/topics/strings.md

```diff
@@ -154,6 +155,90 @@ ${'$'}_9.99
 """
 ```
 
+> To avoid `${'$'}` sequences in strings, you can use the Experimental [multi-dollar string interpolation feature](#multi-dollar-string-interpolation).
+>
+{style="note"}
+
+### Multi-dollar string interpolation
+
+> The multi-dollar string interpolation is [Experimental](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained).
+> It may be changed at any time. We would appreciate your feedback in [YouTrack](https://youtrack.jetbrains.com/issue/KT-2425).
+>
+{style="warning"}
+
+Multi-dollar string interpolation allows you to specify how many consecutive dollar signs are required to trigger interpolation.
```

Comment:
I can, just afraid to make the para too long.

Reply:
True. You could break the paragraph after that point? 

```
Multi-dollar string interpolation allows you to specify how many consecutive dollar signs are required to trigger interpolation.
Interpolation is the process of embedding variables or expressions directly into a string.

While you can [escape literals](#escaped-strings) for single-line strings,
multiline strings in Kotlin don't support backslash escaping.
To include dollar signs (`$`) as literal characters, you must use the `${'$'}` construct to prevent string interpolation.
This approach can make code harder to read, especially when strings contain multiple dollar signs.
```